app/views/forum/topic/form.scala.html

```diff
@@ -1,5 +1,8 @@
 @(categ: lila.forum.Categ, form: Form[_], captcha: lila.common.Captcha)(implicit ctx: Context)
 
+@jQueryTag
```

Comment:
Are you sure?

If I don't have that '@jQueryTag' I get the following error in the console:
"forum-post.js?v=1126:1 Uncaught ReferenceError: $ is not defined".

Do you maybe have to extend / compose from some sort of base template to get jquery?